### PR TITLE
Fix bibtex citation in plot_sensor_regression.py issue#8905

### PR DESCRIPTION
**Pull request by anaradanovic**
#### Reference issue
Example: Fixes issue #8905 insert bibtex citation.

#### What does this implement/fix?
See above, insert bibtex citations.

### CODE REVIEW

**Comment by larsoner** [comment]
On `main` I see:
```
$ git grep Dufau
doc/overview/datasets_index.rst:lexical decision task on 960 English words :footcite:`DufauEtAl2015`. The words
doc/references.bib:@article{DufauEtAl2015,
doc/references.bib:  author = {Dufau, Stéphane and Grainger, Jonathan and Midgley, Katherine J. and Holcomb, Phillip J.},
examples/stats/plot_sensor_regression.py:.. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
mne/datasets/kiloword/kiloword.py:    .. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
tutorials/stats-sensor-space/plot_stats_cluster_erp.py:# .. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
```
Do you want to try changing the others? If not, happy to merge this as is

**Comment by anaradanovic** [comment]
> On `main` I see:
> 
> ```
> $ git grep Dufau
> doc/overview/datasets_index.rst:lexical decision task on 960 English words :footcite:`DufauEtAl2015`. The words
> doc/references.bib:@article{DufauEtAl2015,
> doc/references.bib:  author = {Dufau, Stéphane and Grainger, Jonathan and Midgley, Katherine J. and Holcomb, Phillip J.},
> examples/stats/plot_sensor_regression.py:.. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
> mne/datasets/kiloword/kiloword.py:    .. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
> tutorials/stats-sensor-space/plot_stats_cluster_erp.py:# .. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
> ```
> 
> Do you want to try changing the others? If not, happy to merge this as is

Yup! Working on it right now.

File: examples/stats/plot_sensor_regression.py [hunk 1]
```diff
@@ -10,20 +10,17 @@
...
 
 References
 ----------
-.. [1] Dufau, S., Grainger, J., Midgley, KJ., Holcomb, PJ. A thousand
-   words are worth a picture: Snapshots of printed-word processing in an
-   event-related potential megastudy. Psychological Science, 2015
-.. [2] Hauk et al. The time course of visual word recognition as revealed by
-   linear regression analysis of ERP data. Neuroimage, 2006
+.. footbibliography::
+.. footbibliography::
```
Comment (larsoner) [inline examples/stats/plot_sensor_regression.py:23]:
```suggestion
```
Accidentally added twice

**Comment by rob-luke** [comment]
This file has since been edited in #9068 

@anaradanovic feel free to grab a mentor on discord to run you through how to resolve these conflicts if you need a hand.

**Comment by rob-luke** [comment]
Also a few lines are too long. See https://github.com/mne-tools/mne-python/pull/9058/checks?check_run_id=2114248477#step:7:11

And if you click on the files changed tab above it shows you the lengthy lines

**Comment by britta-wstnr** [comment]
LGTM, merging. Thanks @anaradanovic !

**Pull request merged**